tests/Unit/Core/Addon/Module/ModuleRepositoryTest.php

```diff
@@ -194,7 +194,7 @@ public function test_get_not_enabled_modules()
 
         foreach ($all_modules as $name => $module) {
             // Each installed module must be found in the installed modules list
-            if ($module->attributes->get('productType') == 'module' && $module->database->has('active') && $module->database->get('active') == 0) {
+            if ($module->attributes->get('productType') == 'module' && $module->database->get('installed') == 1  && $module->database->get('active') == 0) {
```

Comment:
ok but why did you add a check on the `installed` attribute? I mean, if the module is active, it must be installed, right ?